### PR TITLE
Post req fix

### LINKED ISSUE #95: Show list of users uploaded items
Similar to the item drawer, but showing only users items

### PR DESCRIPTION
**Pull request by inpatr**
closes #95 
upload works, styled the user items list a bit
sorry for the naming

### CODE REVIEW

File: components/UserListItem/UserListItem.tsx [hunk 1]
```diff
@@ -1,60 +1,131 @@
...
-    requestStatus = <div>No requests</div>;
+type User = {
+  identifier: string;
+  firstname: string;
+  lastname: string;
+};
+
+export const UserListItem = ({ item, i }: Props) => {
+  // ------------- recipient logic ------------ //
+  // let recipientID = item.recipientId;
```
Comment (deankemball) [inline components/UserListItem/UserListItem.tsx:24]: remove commented out code block?

Comment (inpatr) [reply components/UserListItem/UserListItem.tsx:24]: i commented this out for the presentation. it actually worked to show the recipient after a successful trade but somehow it interfered with the filter Buttons.
we could do an extra bugfix-issue for this

**Comment by inpatr** [comment]
this branch should be merged before 84!

**Review by deankemball — COMMENTED** [review]
Image upload and validation works correctly. 
Minor issue in Line 33 on UploadImage.tsx. After attempting an uplaod the file name always has a length so the check for displaying the error will always display the file name concatenated to the error message "INVALID PIC" which looks like this:

image

Could be changed to an alert or something later, not urgent right now.

**Review by dnmct — APPROVED** [review]

**Pull request merged**